### PR TITLE
Addons update

### PR DESCRIPTION
**Pull request by evilnick**
This is still a WIP as I need to do all the steps for the Howto page

### CODE REVIEW

File: pages/k8s/cdk-addons.md [hunk 1]
```diff
@@ -15,10 +15,19 @@ toc: False
...
-and/or to add support for specific features.
+and/or to add support for specific features. These 'addons' have historically 
+been 'baked in' with the default install of Charmed Kubernetes, but as these
+components are no longer going to be updated upstream, these are being replaced by 
+specific **Operator Charms**. Such charms are deployed by Juju into Kubernetes
+itself and require some additional steps. 
 
-This page details these components, their purpose and how they may be
-enabled/disabled where applicable.
+<div class="p-notification--positive is-inline">
```
Comment (evilnick) [inline pages/k8s/cdk-addons.md:24]: N.B. this doesn't display correctly in our preview due to changes in vanilla used in u.c. I will try and sync these at a future date

File: pages/k8s/how-to-addons.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+  title: "How to use CDK add-on Operator Charms"
+  description: Explaining how to install and configure addon operator charms with Charmed Kubernetes.
+keywords: operating, add-ons, addons, config
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: how-to-cdk-addons.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
```
Comment (evilnick) [inline pages/k8s/how-to-addons.md:15]: This page is obviously still very much WIP

File: pages/k8s/cdk-addons.md [hunk 3]
```diff
@@ -34,28 +43,18 @@ with **Charmed Kubernetes**.
...
-since 1.14.
+since 1.14. It will be installed and configured as part of the install
+process of Charmed Kubernetes.
 
-For additional control over CoreDNS, you can also deploy it into the cluster
-using the [CoreDNS Kubernetes operator charm][coredns-charm]. To do so, set
-the `dns-provider` [kubernetes-control-plane configuration][] option to `none` and
-deploy the charm into a Kubernetes model on your cluster. You'll also need
-to cross-model relate it to kubernetes-control-plane:
+If you use the operator charm to deploy CorDNS to your cluster instead,
```
Comment (stonepreston) [inline pages/k8s/cdk-addons.md:49]:
```suggestion
If you use the operator charm to deploy CoreDNS to your cluster instead, 
```

File: pages/k8s/how-to-addons.md [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+juju switch ck-model
+juju consume k8s-model.coredns
+```
+
+The `consume` command is the counterpart to `offer`. It establishes a connection to the specified model and application, and adds that resource to the current model. A running
+application in the model can then be related to it as thought it were a local model 
+resource. In this case you want to connect it to the kubernetes-control-plane:
+
+```bash
+juju relate -m cluster-model coredns kubernetes-control-plane
```
Comment (stonepreston) [inline pages/k8s/how-to-addons.md:209]:
```suggestion
juju relate -m ck-model coredns kubernetes-control-plane
```

**Review by stonepreston — CHANGES_REQUESTED** [review]

File: pages/k8s/how-to-addons.md (hunk 4 above)
Comment (evilnick) [reply pages/k8s/how-to-addons.md:209]: good spot - I thought I'd changed all those!

**Pull request merged**